bundles/org.openhab.binding.sinope/src/main/resources/ESH-INF/i18n/sinope.properties

```diff
@@ -0,0 +1,4 @@
+config-status.error.missing-host-configuration=No host for the sinop� gateway has been provided.
```

Comment:
This file seems copied to old ESH-INF directory by mistake? File already exists in OH-INF.